### PR TITLE
Update Monitor product name

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Updates strings and product word-marks for Monitor behind `SWITCH_MOZILLA_MONITOR_BRAND_NAME`

## Significant changes and points to review

Did I miss anything? 

Note: 

- The product icon is not changing at this time, only the word-mark name.
- I didn't update the annual report pages, since those are a "moment in time"?
- I'm also not updating Monitor URLs just yet, that will come later.

## Issue / Bugzilla link

N/A

## Testing

- [ ] http://localhost:8000/en-US/ (Main navigation)
- [ ] http://localhost:8000/en-US/firefox/accounts/
- [ ] http://localhost:8000/en-US/firefox/browsers/best-browser/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/brave/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/chrome/
- [ ] http://localhost:8000/en-US/firefox/
- [ ] http://localhost:8000/en-US/firefox/ (FF modal)
- [ ] http://localhost:8000/en-US/firefox/privacy/safe-passwords/
- [ ] http://localhost:8000/en-US/firefox/privacy/products/
- [ ] http://localhost:8000/en-US/firefox/privacy/book/
- [ ] http://localhost:8000/en-US/firefox/products/
- [ ] http://localhost:8000/en-US/firefox/welcome/1/
- [ ] http://localhost:8000/en-US/firefox/welcome/7/
- [ ] http://localhost:8000/en-US/firefox/welcome/8/
- [ ] http://localhost:8000/en-US/newsletter/updated/

### CODE REVIEW

**Comment by alexgibson** [comment]
Note: I haven't updated `/welcome/3/` here, as that page is now being removed in https://github.com/mozilla/bedrock/pull/13733

File: l10n/en/firefox/browsers/compare/chrome.ftl [hunk 1]
```diff
@@ -13,10 +13,16 @@ compare-chrome-we-compare-mozilla-firefox = We compare { -brand-name-mozilla } {
 # Main title.
 compare-chrome-comparing-firefox-browser = Comparing { -brand-name-firefox-browser } with { -brand-name-google } { -brand-name-chrome }
 
+# Variables:
+#   $attrs (string) - link to https://monitor.firefox.com with additional attributes for analytics
+compare-chrome-at-firefox-we-have-a-huge-v2 = At { -brand-name-firefox }, we have a huge number of dedicated users who appreciate our steadfast dedication to online privacy. For example, the latest version of { -brand-name-firefox } includes a feature called Enhanced Tracking Protection (ETP) which is turned on by default for all users worldwide. ETP blocks over 2,000 trackers, including social trackers from companies like { -brand-name-facebook }, { -brand-name-twitter }, and LinkedIn. It also has an integrated feature called <a { $attrs }>{ -brand-name-mozilla-monitor }</a> that automatically notifies you if your password has been breached or needs to be updated. In addition to these protections, Private Browsing mode in { -brand-name-firefox } automatically deletes your browsing information such as history and cookies, leaving no trace after you finish your session.
```
Comment (stephaniehobson) [inline l10n/en/firefox/browsers/compare/chrome.ftl:18]: Let's take this opportunity to stop using the `-brand-name-facebook` and `-brand-name-twitter` keywords in the v2 string.

**Review by stephaniehobson — COMMENTED** [review]
I assume we're going to be asked to do all this work for Relay too. I think it would be worth while to bundle it so we're not sending v3 versions of strings for translation only to have v4s in 2 weeks. I've asked Emma to look into it.

**Comment by alexgibson** [comment]
@stephaniehobson thanks, updated.

Ideally, I'd like these strings to get out to localizers soon so we have plenty of time for locales to update. Bunching up translation work makes sense, but I also don't want this to get held up behind a product decision that has yet to be made? Thanks

**Review by stephaniehobson — APPROVED** [review]
Luke says there's no plans to change soon.

R+

**Pull request merged**